000-configure-stats.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# make bash behave
+set -euo pipefail
+IFS=$'\n\t'
+
+if [ -n "${DISABLE_STATS_COLLECTION+set}" ]; then
```

Comment:
Doesn't this make `-e DISABLE_STATS_COLLECTION=false` to also disable stats collection?

Reply:
Yes, but this is following a pretty common bash pattern that if the variable is set at all it is honored. Otherwise we'd need to decide what is false and true; for instance, people might try:

  * `0`
  * `False`
  * `FALSE`
  * `false`
  * `off`
  * `no`

It's easier in this case to just document "if this variable is set at all stats are disabled".

Reply:
Ok. Makes sense.